kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala

```diff
@@ -81,6 +81,9 @@ private[v1] class OperationsResource extends ApiRequestContext with Logging {
       }
       Response.ok().build()
     } catch {
+      case e: KyuubiSQLException =>
+        error(e.getMessage, e)
+        throw e
```

Comment:
also cc @pan3793

Reply:
is that aim to address no error log issue? I also find the same issue and I think we should log it in `RestExceptionMapper` for all exceptions

Reply:
Open #4977 to address it generally.